features/step_definitions/project_steps.rb

```diff
@@ -0,0 +1,207 @@
+    month: "01",
+    year: "2021"
+  )
+end
+
+Given(/^I enter my construction start date$/) do
+  @app.start_construction_date_page.submit(
+    month: "01",
+    year: "2022"
+  )
+end
+
+Given(/^I enter my ready for service date$/) do
+  @app.ready_for_service_date_page.submit(
+    month: "01",
+    year: "2023"
+  )
+end
+
+Given(/^I add my project risks$/) do
+  @app.proposal_overview_page.add_risks.click
+  @app.risks_page.submit(
+    risks: %w[tidal_flooding sea_flooding]
+  )
+  # Picks the first risk as the main risk - else picks the last risk
+  @app.main_risk_page.submit(
+    choose_first: true
+  )
+
+  @app.flood_protection_outcomes_page.submit(
+    a1: "1000",
+    a2: "100",
+    a3: "10",
+    b1: "1000",
+    b2: "100",
+    b3: "10",
+    c1: "1000",
+    c2: "100",
+    c3: "10"
+  )
+end
+
+Given(/^I return to the overview page$/) do
+  @app.proposal_under_review_page.return_to_the_proposal_overview_page.click
+end
+
+Given(/^I revert to draft$/) do
+  @app.proposal_overview_page.revert_to_draft.click
+end
+
+Given(/^I add the standard of protection before project starts as "([^"]*)"$/) do |before|
+  @app.proposal_overview_page.add_standard_of_protection.click
+  @app.standard_of_protection_page.submit(
+    option: before.to_sym
+  )
+end
+
+Given(/^I add the standard of protection after project completes as "([^"]*)"$/) do |after|
+  @app.standard_of_protection_after_page.submit(
+    option: after.to_sym
+  )
+end
+
+Given(/^I enter the projects goal approach$/) do
+  @app.proposal_overview_page.add_goals.click
+  @app.approach_page.submit(
+    approach: "This is the projects approach...."
+  )
+end
+
+# rubocop:disable Metrics/BlockLength
+Given(/^I enter environmental outcomes improvements$/) do
+  @app.proposal_overview_page.add_environmental_outcomes.click
+  @app.environmental_outcomes_page.submit(
+    improve: true
+  )
+  @app.surface_and_groundwater_amount_page.submit(
+    length: "1000"
+  )
+  @app.improve_spa_or_sac_page.submit(
+    improve: true
+  )
+  @app.improve_habitat_amount_page.submit(
+    amount: "2000"
+  )
+  @app.improve_river_page.submit(
+    improve: true
+  )
+  @app.improve_river_amount_page.submit(
+    amount: "3000"
+  )
+  @app.habitat_creation_page.submit(
+    create: true
+  )
+  @app.habitat_creation_amount_page.submit(
+    amount: "4000"
+  )
+  @app.remove_fish_barrier_page.submit(
+    remove: true
+  )
+  @app.remove_eel_barrier_page.submit(
+    remove: true
+  )
+  @app.fish_or_eel_amount_page.submit(
+    amount: "5000"
+  )
+end
+# rubocop:enable Metrics/BlockLength
+
+Then(/^I upload my project funding calculator file "([^"]*)"$/) do |filename|
+  @app.proposal_overview_page.add_funding_calculator.click
+  @app.funding_calculator_page.submit(
+    file: filename
+  )
+  @app.funding_calculator_summary_page.submit
+end
+
+Then(/^I should see my entered details in the the proposal overview$/) do
+  @project_number = @app.proposal_overview_page.project_number.text
```

Comment:
Is this not used?

Reply:
Commented out for the moment as it will be used in future tests